lib/virt_autotest/common.pm

```diff
@@ -111,7 +125,7 @@ if (get_var("REGRESSION", '') =~ /xen/) {
     );
     # Filter out guests not allowed for the detected SLE version
     if (is_sle('=12-SP5')) {
-        my @allowed_guests = qw(sles12sp5HVM sles12sp5PV sles15sp5HVM sles15sp5PV sles15sp6HVM sles15sp6PV);
+        my @allowed_guests = qw(sles12sp5HVM sles12sp5PV sles12sp5ltssesHVM sles12sp5ltssesPV sles15sp5HVM sles15sp5PV sles15sp6HVM sles15sp6PV);
```

Comment:
I am afraid, here differentiation is needed, as given in ticket:
if  (get_var('EXTENDED_SECURITY'))  {
  @allowed_guests = qw(sles12sp5ltssesHVM sles12sp5ltssesPV sles15sp5HVM sles15sp5PV sles15sp6HVM sles15sp6PV);
} else
  my @allowed_guests = qw(sles12sp5HVM sles12sp5PV sles15sp5HVM sles15sp5PV sles15sp6HVM sles15sp6PV);
}